templates/consoleReadmeSuffix.md

```diff
@@ -0,0 +1,39 @@
+## Getting started
```

Comment:
Let's use `snake_case` in filenames instead of `camelCase`, as per convention:
https://www.notion.so/apify/Coding-standards-editor-setup-58912bbe2c0c4ad9961ae818179bcbd9?pvs=4#9f86cbe9e80d46e197414ca1ddba1046

Reply:
Lol, good catch. I prefer dash case, but we can universaly agree that camel case is wrong for file names 😅